File: JavaScript/js/exercicioManzandoPg46J.js
function mediaPares(){
    let contador = 50
    let soma = 0;
    let pares = 0
    // while(contador <= 70){
    //     soma += contador
    //     media = soma / 11 
    //     contador += 2
    // } 

    while(contador <= 70){
        if(contador%2==0){
            soma += contador
            pares++
        }
        contador++
    }
    let media = soma / pares
    
    document.getElementById("mostra").innerHTML= "A soma dos valores pares de 50 ate 70 é "+soma+"<br> A média dos números é "+media
}

File: Java/manzando/exercicioManzandoPg46J.java
package manzando;

public class exercicioManzandoPg46J {
	public static void main(String[] args) {
		int contador = 50, soma = 0, pares = 0;
		
		while(contador <= 70){
	        if(contador%2==0){
	            soma += contador;
	            pares++;
	        }
	        contador++;
	    }
		
	    int media = soma / pares;
	    
	    System.out.println("A soma dos valores pares de 50 ate 70 é "+soma+"\nA média dos números é "+media);
	}
}
